fpga_interchange/macros.cc

```diff
@@ -78,6 +79,7 @@ void Arch::expand_macros()
 
             // Get the ultimate root of this macro expansion
             IdString parent = (cell->macro_parent == IdString()) ? cell->name : cell->macro_parent;
+            log_info("%s %s\n", cell->name.c_str(ctx), parent.c_str(ctx));
```

Comment:
this looks like debugging that should either be removed or have some descriptive text added and a `ctx->debug` gate

Reply:
Removed